src/neo-vm/OpCode.cs

```diff
@@ -225,9 +225,18 @@ public enum OpCode : byte
         /// Pop the address of a function from the stack, and call the function.
         /// </summary>
         CALLA = 0x36,
-        THROW = 0x37,
-        THROWIF = 0x38,
-        THROWIFNOT = 0x39,
+        /// <summary>
+        /// It turns the vm state to FAULT immediately, and cannot be caught.
+        /// </summary>
+        ABORT = 0x37,
+        /// <summary>
+        /// Pop the top value of the stack, if it false, then exit vm execution and set vm state to FAULT.
+        /// </summary>
+        ASSERT = 0x38,
+        /// <summary>
+        /// It turns the vm state to FAULT immediately, but it can be caught.
+        /// </summary>
+        THROW = 0x3A,
```

Comment:
0x39?

Reply:
I think `THROW` is more close to `TRY` and `CATCH`.

Reply:
I prefer this order of instructions:
```
THROW
// TRY - ENDF
ASSERT
ABORT
RET
```